src/static/css/page.css

```diff
@@ -44,7 +43,7 @@
   grid-area: index;
 }
 .index .index-box {
-  margin: 0 8px 8px;
+  margin: 20px 8px 8px;
```

Comment:
This top 20px matches the pages' `h1 class="title"`, so consistent top margin everywhere.

Reply:
Not sure about this, I think we want all titles to be on the top of the container. I'll need to take a closer look.

Reply:
This comment I added is related to the main menu pages: Table of Contents, Contributors & Methodology. now have a consistent top margin, compared to the main header.

Sure, any feedback is appreciated. Let me know if you have any questions.